### PR TITLE
Local server using Google Cloud SQL

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR adds a new module - `local-cloud-sql` with a local server, which uses Cloud SQL.

A process of the preliminary configuration described in the `README.md`.

Other changes:
- `local-jdbc` module was renamed to `local-my-sql`.
- Initialization of IDs for `MyListViewProjection` and `DraftTasksViewProjection` was changed to support a correct work with the new server.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: deployment/local-cloud-sql/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+##### Cloud Platform project
+
+* [Select or create](https://console.cloud.google.com/cloud-resource-manager?_ga=2.85931665.-903606831.1506333808)
+    a Cloud Platform project.
+* [Enable](https://support.google.com/cloud/answer/6293499#enable-billing) billing for your project.
+
+##### Cloud SQL
+
+* [Enable](https://console.cloud.google.com/flows/enableapi?apiid=sqladmin&redirect=https:%2F%2Fconsole.cloud.google.com&_ga=2.93339541.-903606831.1506333808) the Cloud SQL Administration API.
```
Comment (armiol) [inline deployment/local-cloud-sql/README.md]: Let's review all these links and drop the Google Analytics anchor. That's something specific to your session and our console. 

In this case the part to drop is `&_ga=2.93339541.-903606831.1506333808`. Please do that for the links and verify they still work by going to browser incognito mode.

File: deployment/local-cloud-sql/build.gradle [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+dependencies {
+    compile group: 'io.spine', name: 'jdbc-rdbms', version: spineVersion
+    compile 'com.google.cloud.sql:mysql-socket-factory:1.0.4'
```
Comment (armiol) [inline deployment/local-cloud-sql/build.gradle]: I think this version should also go to `ext.gradle` or some place like that. The idea is to have all the versions that change often (along with the updates of the Google Cloud services) externalized.

File: deployment/local-cloud-sql/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# local-cloud-sql
+
+A local server, which uses Google Cloud SQL.
```
Comment (armiol) [inline deployment/local-cloud-sql/README.md]: Uses for what?

Please explain how it works in more details.

**Pull request merged**